install/kubernetes/cilium/charts/nodeinit/values.yaml

```diff
@@ -2,6 +2,10 @@
 # managed by Cilium (GKE)
 restartPods: false
```

Comment:
Couldn't we reuse this same option for EKS? We would need to add EKS in the previous line.

Reply:
This seems better to me, rather than having individual pods restart options for each platform.

Next question is if it's easy to detect that it's an EKS environment, is there some way we can reliably detect this to run the special restart logic introduced in this PR?

Reply:
I was looking into it a bit, best thing I've found so far is `/etc/cni/net.d/10-aws.conflist`, which is created by the EKS "aws-node" daemonset.

Reply:
Also quick question; some of the former commits don't have the sign off, will this be a problem? I didn't see any info regarding that on the contributing page.

Reply:
@tom-handlaw-hs



> Also quick question; some of the former commits don't have the sign off, will this be a problem? I didn't see any info regarding that on the contributing page.

Yes, you can amend previous commits with `git rebase -i master` and select `reword` for those particular commits. Once you are editing them you can add the sign-off message.